.travis.yml

```diff
@@ -1,21 +1,30 @@
 language: python
 
+dist: xenial
+
 services:
-  - rabbitmq
+  - docker
+
+before_install:
+  - docker run -d --rm -p 15672:15672 -p 5672:5672 -p 5671:5671 --name nameko-rabbitmq nameko/nameko-rabbitmq:3.6.6
 
 python:
-  - '3.4'
   - '3.5'
   - '3.6'
-  - nightly
+  - '3.7'
 
-matrix:
-  allow_failures:
-    - python: nightly
+env:
+  - NAMEKO_VERSION===2.8.5
```

Comment:
Might be worth quoting these. It's easy to miss the fact that there's three `=` signs

Reply:
Even clearer, and allows changing more than one if needed:

```
env:
    DEPS="nameko==2.9.1"
install:
    pip install -U $DEPS
```

Reply:
That's nicer! Thanks :+1:

Reply:
Done, merging ...)